### PR TITLE
Adds Ethari

### PR DESCRIPTION
**Pull request by FlattestGuitar**
We're not a furry ERP server, I swear.

Everything that #4 had, plus a few tweaks to sprites, no "Awoo" and "Kek" in the namelist and all instances of "fox" changed to "Ethari", because we're hard scifi ok

### CODE REVIEW

File: code/__HELPERS/mobs.dm [hunk 1]
```diff
@@ -52,12 +52,18 @@
...
 		init_sprite_accessory_subtypes(/datum/sprite_accessory/snouts, GLOB.snouts_list)
+	if(!GLOB.snouts_ethari_list.len)
+		init_sprite_accessory_subtypes(/datum/sprite_accessory/snouts_ethari, GLOB.snouts_ethari_list)
+	if(!GLOB.ears_ethari_list.len)
+		init_sprite_accessory_subtypes(/datum/sprite_accessory/anthro_ears, GLOB.ears_ethari_list)
 	if(!GLOB.horns_list.len)
 		init_sprite_accessory_subtypes(/datum/sprite_accessory/horns, GLOB.horns_list)
 	if(!GLOB.ears_list.len)
-		init_sprite_accessory_subtypes(/datum/sprite_accessory/ears, GLOB.horns_list)
+		init_sprite_accessory_subtypes(/datum/sprite_accessory/ears, GLOB.ears_list)
```
Comment (FlattestGuitar) [inline code/__HELPERS/mobs.dm]: This was someone else's error; irrelevant to the PR but I fixed it while I was there.

File: code/modules/client/preferences.dm [hunk 2]
```diff
@@ -228,7 +228,7 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 
 			if(HAIR in pref_species.species_traits)
 
-				dat += "<td valign='top' width='21%'>"
+				dat += "<td valign='top' width='10%'>"
```
Comment (FlattestGuitar) [inline code/modules/client/preferences.dm]: All these `%` changes change how this stuff ends up looking in the character setup menu. The more width is has the larger it is, duh. Some of the pre-existing stuff had width values far too large for the content they had, making the new stuff have to squish up in a corner. Not good.

File: code/modules/client/preferences.dm [hunk 3]
```diff
@@ -981,7 +1011,7 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 						var/temp_hsv = RGBtoHSV(new_mutantcolor)
 						if(new_mutantcolor == "#000000")
 							features["mcolor"] = pref_species.default_color
-						else if((MUTCOLORS_PARTSONLY in pref_species.species_traits) || ReadHSV(temp_hsv)[3] >= ReadHSV("#7F7F7F")[3]) // mutantcolors must be bright, but only if they affect the skin
+						else if((MUTCOLORS_PARTSONLY in pref_species.species_traits) || ReadHSV(temp_hsv)[3] >= ReadHSV("#252525")[3]) // mutantcolors must be bright, but only if they affect the skin
```
Comment (FlattestGuitar) [inline code/modules/client/preferences.dm]: There's a lower limit on how dark characters' skin can get. No idea why. I lowered it to allow for mostly black characters.

File: code/modules/food_and_drinks/food/snacks/meat.dm [hunk 4]
```diff
@@ -87,6 +87,13 @@
 	tastes = list("meat" = 4, "scales" = 1)
 	foodtype = MEAT | RAW
 
+/obj/item/reagent_containers/food/snacks/meat/slab/human/mutant/fox
+	icon_state = "foxmeat"
+	desc = "Rawr?"
```
Comment (FlattestGuitar) [inline code/modules/food_and_drinks/food/snacks/meat.dm]: It's literally a sliced up piece of a furry. There's nothing cute about this. Don't :eyes: me.

File: code/modules/mob/living/carbon/human/species.dm [hunk 5]
```diff
@@ -415,7 +415,7 @@
 
 /datum/species/proc/handle_mutant_bodyparts(mob/living/carbon/human/H, forced_colour)
 	var/list/bodyparts_to_add = mutant_bodyparts.Copy()
-	var/list/relevent_layers = list(BODY_BEHIND_LAYER, BODY_ADJ_LAYER, BODY_FRONT_LAYER)
+	var/list/relevant_layers = list(BODY_BEHIND_LAYER, BODY_ADJ_LAYER, BODY_FRONT_LAYER)
```
Comment (FlattestGuitar) [inline code/modules/mob/living/carbon/human/species.dm]: War crimes.

File: code/modules/reagents/chemistry/recipes/others.dm [hunk 6]
```diff
@@ -139,6 +139,12 @@
 	results = list("lizardmutationtoxin" = 1)
 	required_reagents = list("unstablemutationtoxin" = 1, "radium" = 1) //mutant
 
+/datum/chemical_reaction/ethari_mutation_toxin
+	name = "Ethari Mutation Toxin"
+	id = "etharimutationtoxin"
+	results = list("etharimutationtoxin" = 1)
+	required_reagents = list("unstablemutationtoxin" = 1, "welding_fuel" = 1)
```
Comment (FlattestGuitar) [inline code/modules/reagents/chemistry/recipes/others.dm]: A different recipe, perhaps? Most of the stuff in similar ones isn't hard to obtain at all.

File: code/modules/food_and_drinks/food/snacks/meat.dm [hunk 7]
```diff
@@ -87,6 +87,13 @@
...
+	filling_color = "#DF8134"
+	tastes = list("meat" = 4, "stray hairs" = 1)
```
Comment (SamHPurp) [inline code/modules/food_and_drinks/food/snacks/meat.dm]: ew

File: code/modules/language/canilunzt.dm [hunk 8]
```diff
@@ -0,0 +1,15 @@
+/datum/language/canilunzt
+	name = "Canilunzt"
+	desc = "The common language of Ethari, composed of low growls and yelps."
+	speech_verb = "growls"
+	ask_verb = "growls"
+	exclaim_verb = "roars"
+	key = "7"
+	flags = TONGUELESS_SPEECH
```
Comment (SamHPurp) [inline code/modules/language/canilunzt.dm]: erm, this is a wierdly named flag - what does it actually do?

Comment (FlattestGuitar) [reply code/modules/language/canilunzt.dm]: Means you can speak it without a tongue. You still have issues with pronouncing most letters, but you can speak the language.

File: code/modules/mob/living/carbon/human/species_types/ethari.dm [hunk 9]
```diff
@@ -0,0 +1,38 @@
...
+	// Furry people with bushy tails and stuff.
+	name = "Ethari"
+	id = "ethari"
+	default_color = "#DF8134"
+	species_traits = list(MUTCOLORS,EYECOLOR,LIPS,HAIR)
+	mutant_bodyparts = list("tail_ethari", "body_markings", "snout_ethari", "ears_ethari")
+	coldmod = 0.67
+	heatmod = 1.5
+	brutemod = 1.2
+	burnmod = 1.2
```
Comment (Birdtalon) [inline code/modules/mob/living/carbon/human/species_types/ethari.dm]: Do we definitely want 20% extra burn and brute for these guys? What's the reasoning behind it?

Comment (SamHPurp) [reply code/modules/mob/living/carbon/human/species_types/ethari.dm]: fukken furries reeeee

is the reasoning

File: code/modules/language/canilunzt.dm (hunk 8 above)
Comment (SamHPurp) [reply code/modules/language/canilunzt.dm]: seems goofy

File: code/modules/mob/living/carbon/human/species_types/ethari.dm (hunk 9 above)
Comment (FlattestGuitar) [reply code/modules/mob/living/carbon/human/species_types/ethari.dm]: ^ more or less. Won't have people complaining about snowflakes if they're clearly inferior. I don't think it's that impactful, considering 95% of the game doesn't involve bashing someone's skull in, but when it comes to it, it's a weakness you have to consider.

**Pull request merged**